google-cloud-vision/src/test/java/com/google/cloud/vision/it/ITSystemTest.java

```diff
@@ -500,7 +500,10 @@ public void detectWebEntitiesIncludeGeoResultsTest() throws IOException {
         actual.add(entity.getDescription());
       }
     }
-    assertThat(actual).contains("Skyscraper");
+    List<String> expectedResults = new ArrayList<>();
+    expectedResults.add("Skyscraper");
+    expectedResults.add("Suburb");
```

Comment:
nit: you could make this a one-liner with Guava's `Lists.newArrayList("string1", "string2")`